src/models.ts

```diff
@@ -59,12 +59,41 @@ export const isJSONRPCResponses = (
   return Array.isArray(payload) && payload.every(isJSONRPCResponse);
 };
 
-export interface JSONRPCError {
+export interface JSONRPCErrorObject {
```

Comment:
Oh, I know I suggested `JSONRPCError` to be the name of the `Error` class, but I totally forgot we already use the name for something else. Can you please keep this as `JSONRPCError` to keep backward compatibility?

I'm not sure what to call the `Error` class yet, but I'll sleep on it. Let me know if you have some suggestions too!

Reply:
Alright, will change the name when a new one is decided. I had thought it might be better to keep the class name simpler since it would probably be used more but I can see the backwards compatibility is more of a concern. Perhaps for a new name something like `JSONRPCErrorParam` could be used or maybe just switch the names so the class becomes `JSONRPCErrorObject` since the [specification](https://www.jsonrpc.org/specification#error_object) refers to it as "Error object"?

Reply:
I have changed `JSONRPCError` back to what it was. I have updated the error class name to your suggestion of `JSONRPCErrorException` for now since that looks like the best suggestion so far.